<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>8</number>
    </property>
    <item row="0" column="0">
     <widget class="GcodeView" name="graphicsView"/>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSlider" name="LayerHeight">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0" columnstretch="0,0,0,3,0,0,0,0,0,0">
      <item row="2" column="1">
       <widget class="QPushButton" name="panUp">
        <property name="text">
         <string>Pan Up</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="panLeft">
        <property name="text">
         <string>Pan Left</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="panDown">
        <property name="text">
         <string>Pan Down</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="6">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="6">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="9">
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="9">
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="9">
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="7">
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="5">
       <spacer name="horizontalSpacer_16">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="5">
       <spacer name="horizontalSpacer_17">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="5">
       <spacer name="horizontalSpacer_18">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="8">
       <spacer name="horizontalSpacer_22">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="8">
       <spacer name="horizontalSpacer_23">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="8">
       <spacer name="horizontalSpacer_24">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="zoomIn">
        <property name="text">
         <string>Zoom In</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="zoomOut">
        <property name="text">
         <string>Zoom Out</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="panRight">
        <property name="text">
         <string>Pan Right</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <spacer name="horizontalSpacer_19">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="7">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEditConfigFile"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Config file:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="buttonConfigBrowse">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>3D model:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="lineEdit3dModelFile"/>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="button3dModelBrowse">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="buttonSlice">
        <property name="text">
         <string>slice!</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_task">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPathMonger">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionLoad_GCode"/>
    <addaction name="actionExport_Gcode_File"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All"/>
    <addaction name="actionClose_All_Except"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <addaction name="menuPathMonger"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_GCode">
   <property name="text">
    <string>Open gcode file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClose_All_Except">
   <property name="text">
    <string>Close All Except</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionExport_Gcode_File">
   <property name="text">
    <string>Slice 3D model...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSlice_3D_model">
   <property name="text">
    <string>FAKE</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GcodeView</class>
   <extends>QGraphicsView</extends>
   <header>gcodeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
